package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "fastify-demo",
+  "version": "0.0.0",
+  "description": "The official Fastify demo!",
+  "main": "app.js",
+  "type": "module",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "build": "rm -rf dist && tsc",
+    "watch": "npm run build -- --watch",
+    "test": "tap test/**/*.ts",
```

Comment:
I think it can be a good idea to run test with `tap`, it handle transpilation and code coverage for us.